conjure-go-client/httpclient/config.go

```diff
@@ -106,6 +106,7 @@ type SecurityConfig struct {
 	CAFiles  []string `json:"ca-files,omitempty" yaml:"ca-files,omitempty"`
 	CertFile string   `json:"cert-file,omitempty" yaml:"cert-file,omitempty"`
 	KeyFile  string   `json:"key-file,omitempty" yaml:"key-file,omitempty"`
+	CAs      []string `json:"cas"`
```

Comment:
+1 to @bmoylan comment. It's probably also worth touching on order precedence.
Also can you split this into it's own PR since it's unrelated to configuring the `InsecureSkipVerify`